testing/src/testing.cpp

```diff
@@ -23,10 +23,28 @@
 namespace pika { namespace util {
     namespace detail {
 
+        std::atomic<std::size_t> fixture::sanity_tests_(0);
         std::atomic<std::size_t> fixture::sanity_failures_(0);
+        std::atomic<std::size_t> fixture::test_tests_(0);
         std::atomic<std::size_t> fixture::test_failures_(0);
 
-        void fixture::increment(counter_type c)
+        void fixture::increment_tests(counter_type c)
+        {
+            switch (c)
+            {
+            case counter_sanity:
+                ++sanity_tests_;
+                return;
+            case counter_test:
+                ++test_tests_;
+                return;
+            default:
+                break;
+            }
+            PIKA_ASSERT(false);
```

Comment:
can the `PIKA_ASSERT(false)` be in the `default:`?

Reply:
Yes, I think so.

Reply:
I did not update this yet...